### PR TITLE
docs: (IAC-958) update DAC SingleStore.md to include iac-aws and oss-k8s

### PR DESCRIPTION
**Pull request by dhoucgitter**
# Changes
The DAC SingleStore.md has a [section](https://github.com/sassoftware/viya4-deployment/blob/bc2992bc6f79630678d259a1fcfbf79a2bb18e14/docs/user/SingleStore.md#deploying-sas-viya-with-singlestore-using-sas-viya-4-deployment) for Deploying SAS Viya with SingleStore Using SAS Viya 4 Deployment.    It mentions using viya4-iac-azure.  We need to add viya4-iac-aws and viya4-iac-k8s.

### CODE REVIEW

**Review by jarpat — APPROVED** [review]

**Review by riragh — APPROVED** [review]

File: docs/user/SingleStore.md [hunk 1]
```diff
@@ -8,9 +8,27 @@ If your SAS software order included SAS Viya with SingleStore, additional requir
...
 ## Deploying SAS Viya with SingleStore Using SAS Viya 4 Deployment
 
-You can deploy SAS Viya with SingleStore into a Kubernetes cluster that is running in Microsoft Azure. The [SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure](https://github.com/sassoftware/viya4-iac-azure) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments. 
+You can deploy SAS Viya with SingleStore into a Kubernetes cluster running under:
+- Azure Kubernetes Service (AKS) in Microsoft Azure
+- Elastic Kubernetes Service (EKS) in Amazon Web Services
+- Open Source Kubernetes on your own machines
 
-Refer to the [SingleStore sample input file](https://github.com/sassoftware/viya4-iac-azure/blob/main/examples/sample-input-singlestore.tfvars) for Terraform configuration values that create an Azure cluster suitable for deploying SAS Viya and SingleStore.
+## Cluster provisioning for SAS Viya with SingleStore
```
Comment (supear) [inline docs/user/SingleStore.md:16]: Provisioning

File: docs/user/SingleStore.md [hunk 2]
```diff
@@ -8,9 +8,27 @@ If your SAS software order included SAS Viya with SingleStore, additional requir
...
+
+### Azure Kubernetes Service Cluster in Microsoft Azure
```
Comment (supear) [inline docs/user/SingleStore.md:18]: Azure Kubernetes Service (AKS) Cluster in Microsoft Azure

File: docs/user/SingleStore.md [hunk 3]
```diff
@@ -8,9 +8,27 @@ If your SAS software order included SAS Viya with SingleStore, additional requir
...
+
+The [SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure](https://github.com/sassoftware/viya4-iac-azure) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments. 
+Refer to the [SingleStore sample input file](https://github.com/sassoftware/viya4-iac-azure/blob/main/examples/sample-input-singlestore.tfvars) for Terraform configuration values that create an AKS cluster suitable for deploying SAS Viya and SingleStore.
+
+### Elastic Kubernetes Service Cluster in Amazon Web Services
```
Comment (supear) [inline docs/user/SingleStore.md:23]: EKS Cluster in AWS

File: docs/user/SingleStore.md [hunk 4]
```diff
@@ -8,9 +8,27 @@ If your SAS software order included SAS Viya with SingleStore, additional requir
 
 ## Deploying SAS Viya with SingleStore Using SAS Viya 4 Deployment
 
-You can deploy SAS Viya with SingleStore into a Kubernetes cluster that is running in Microsoft Azure. The [SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure](https://github.com/sassoftware/viya4-iac-azure) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments. 
+You can deploy SAS Viya with SingleStore into a Kubernetes cluster running under:
+- Azure Kubernetes Service (AKS) in Microsoft Azure
+- Elastic Kubernetes Service (EKS) in Amazon Web Services
```
Comment (supear) [inline docs/user/SingleStore.md:13]: Elastic Kubernetes Service (EKS) in Amazon Web Services (AWS)

Comment (dhoucgitter) [reply docs/user/SingleStore.md:13]: added (AWS)

File: docs/user/SingleStore.md (hunk 1 above)
Comment (dhoucgitter) [reply docs/user/SingleStore.md:16]: changed

File: docs/user/SingleStore.md (hunk 2 above)
Comment (dhoucgitter) [reply docs/user/SingleStore.md:18]: added

File: docs/user/SingleStore.md [hunk 5]
```diff
@@ -8,9 +8,27 @@ If your SAS software order included SAS Viya with SingleStore, additional requir
...
+- Open Source Kubernetes on your own machines
 
-Refer to the [SingleStore sample input file](https://github.com/sassoftware/viya4-iac-azure/blob/main/examples/sample-input-singlestore.tfvars) for Terraform configuration values that create an Azure cluster suitable for deploying SAS Viya and SingleStore.
+## Cluster provisioning for SAS Viya with SingleStore
+
+### Azure Kubernetes Service Cluster in Microsoft Azure
+
+The [SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure](https://github.com/sassoftware/viya4-iac-azure) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments.
```
[the remark this answers is not in the document]
Comment (dhoucgitter) [reply docs/user/SingleStore.md:20]: updated

File: docs/user/SingleStore.md (hunk 3 above)
Comment (dhoucgitter) [reply docs/user/SingleStore.md:23]: updated

File: docs/user/SingleStore.md [hunk 6]
```diff
@@ -8,9 +8,27 @@ If your SAS software order included SAS Viya with SingleStore, additional requir
...
+
+### Azure Kubernetes Service Cluster in Microsoft Azure
+
+The [SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure](https://github.com/sassoftware/viya4-iac-azure) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments. 
+Refer to the [SingleStore sample input file](https://github.com/sassoftware/viya4-iac-azure/blob/main/examples/sample-input-singlestore.tfvars) for Terraform configuration values that create an AKS cluster suitable for deploying SAS Viya and SingleStore.
+
+### Elastic Kubernetes Service Cluster in Amazon Web Services
+
+The [SAS Viya 4 Infrastructure as Code (IaC) for Amazon Web Services (AWS)](https://github.com/sassoftware/viya4-iac-aws) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments. 
+Refer to the [SingleStore sample input file](https://github.com/sassoftware/viya4-iac-aws/blob/main/examples/sample-input-singlestore.tfvars) for Terraform configuration values that create an EKS cluster suitable for deploying SAS Viya and SingleStore.
```
[the remark this answers is not in the document]
Comment (dhoucgitter) [reply docs/user/SingleStore.md:26]: applied

File: docs/user/SingleStore.md [hunk 7]
```diff
@@ -8,9 +8,27 @@ If your SAS software order included SAS Viya with SingleStore, additional requir
...
+- Elastic Kubernetes Service (EKS) in Amazon Web Services
+- Open Source Kubernetes on your own machines
 
-Refer to the [SingleStore sample input file](https://github.com/sassoftware/viya4-iac-azure/blob/main/examples/sample-input-singlestore.tfvars) for Terraform configuration values that create an Azure cluster suitable for deploying SAS Viya and SingleStore.
+## Cluster provisioning for SAS Viya with SingleStore
+
+### Azure Kubernetes Service Cluster in Microsoft Azure
+
+The [SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure](https://github.com/sassoftware/viya4-iac-azure) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments. 
+Refer to the [SingleStore sample input file](https://github.com/sassoftware/viya4-iac-azure/blob/main/examples/sample-input-singlestore.tfvars) for Terraform configuration values that create an AKS cluster suitable for deploying SAS Viya and SingleStore.
```
[the remark this answers is not in the document]
Comment (dhoucgitter) [reply docs/user/SingleStore.md:21]: applied

File: docs/user/SingleStore.md [hunk 8]
```diff
@@ -8,9 +8,27 @@ If your SAS software order included SAS Viya with SingleStore, additional requir
...
+
+### Elastic Kubernetes Service Cluster in Amazon Web Services
+
+The [SAS Viya 4 Infrastructure as Code (IaC) for Amazon Web Services (AWS)](https://github.com/sassoftware/viya4-iac-aws) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments. 
+Refer to the [SingleStore sample input file](https://github.com/sassoftware/viya4-iac-aws/blob/main/examples/sample-input-singlestore.tfvars) for Terraform configuration values that create an EKS cluster suitable for deploying SAS Viya and SingleStore.
+
+### Open Source Kubernetes Cluster
+
+The [SAS Viya 4 Infrastructure as Code (IaC) for Open Source Kubernetes](https://github.com/sassoftware/viya4-iac-k8s) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments. 
+Refer to the [SingleStore sample input file](https://github.com/sassoftware/viya4-iac-k8s/blob/main/examples/vsphere/sample-terraform-static-singlestore.tfvars) for Terraform configuration values that create an Open Source Kubernetes cluster suitable for deploying SAS Viya and SingleStore.
```
[the remark this answers is not in the document]
Comment (dhoucgitter) [reply docs/user/SingleStore.md:31]: applied

File: docs/user/SingleStore.md [hunk 9]
```diff
@@ -8,9 +8,27 @@ If your SAS software order included SAS Viya with SingleStore, additional requir
...
+## Cluster provisioning for SAS Viya with SingleStore
+
+### Azure Kubernetes Service Cluster in Microsoft Azure
+
+The [SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure](https://github.com/sassoftware/viya4-iac-azure) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments. 
+Refer to the [SingleStore sample input file](https://github.com/sassoftware/viya4-iac-azure/blob/main/examples/sample-input-singlestore.tfvars) for Terraform configuration values that create an AKS cluster suitable for deploying SAS Viya and SingleStore.
+
+### Elastic Kubernetes Service Cluster in Amazon Web Services
+
+The [SAS Viya 4 Infrastructure as Code (IaC) for Amazon Web Services (AWS)](https://github.com/sassoftware/viya4-iac-aws) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments.
```
[the remark this answers is not in the document]
Comment (dhoucgitter) [reply docs/user/SingleStore.md:25]: updated

**Comment by dhoucgitter** [comment]
@supear, changes applied, could you please confirm that I didn't miss any of your suggested updates, thanks.

**Review by supear — APPROVED** [review]

**Review by thpang — APPROVED** [review]

**Pull request merged**